.github/workflows/docker-builds.yml

```diff
@@ -22,13 +22,16 @@ concurrency:
 
 jobs:
   docker-build:
-    runs-on: [self-hosted, linux.2xlarge]
+    runs-on: [self-hosted, linux.2xlarge, linux.rocm.gpu.mi300.8]
```

Comment:
@akashveramd Please read the documentation for `runs-on`: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on:~:text=If%20you%20specify%20an%20array%20of%20strings%20or%20variables%2C%20your%20workflow%20will%20execute%20on%20any%20runner%20that%20matches%20all%20of%20the%20specified%20runs%2Don%20values.